README.md

```diff
@@ -204,13 +213,17 @@ Returns the C. elegans connectome represented as a [NetworkX](http://networkx.gi
 
 ```
 
-Modelling data
+Modeling data
 --------------
 
-PyOpenWorm also provides access to store and retrieve data about models.
-Following are some examples of these types of operations.
+As described above, ultimately, ion channel models will be part of the ChannelWorm
```

Comment:
This is a bit overwrought. We established above that biological metadata, including about ion channels, is included the DB. We don't want to discuss the current state of the project and future plans, just, "This is what PyOpenWorm is for, and this is how you use it"

Reply:
Again, can definitely take this out, but I think it is doubly confusing that PyOW has ion channel models, considering that we have a separate project for that.  At least if PyOW only had other types of information we could say that ChannelWorm is for ion channel models and PyOW happens to have a few scattered facts, but ion channels are the core use case for CW.  Again, happy to take it out, but just wanted to make that point clear so we could discuss it.